src/routes/category/[category]/+page.svelte

```
@@ -0,0 +1,34 @@
+<script lang="ts">
+    import ArticleList from '$lib/components/ui/ArticleList.svelte';
+    import RelatedCategories from '$lib/components/ui/RelatedCategories.svelte';
```

ESLint

[error] 'RelatedCategories' is defined but never used.

(@typescript-eslint/no-unused-vars)